compiler/rustc_serialize/src/leb128.rs

```diff
@@ -1,3 +1,4 @@
+use crate::int_overflow::DebugStrictAdd;
```

Comment:
Nit: same, would be good to link to the reported issue so we have some idea what benchmarks regressed in the future to evaluate if it's still needed.